core/system/src/export_table.c

```diff
@@ -145,5 +145,15 @@ const TUvisorExportTable __uvisor_export_table = {
         .thread_destroy = thread_destroy,
         .thread_switch = thread_switch,
     },
+    .pool_queue = {
+        .pool_init = uvisor_pool_init,
```

Comment:
@Patater : in the pool_queue - istn't it over the top to use pool_* prefixes for each member ?